Resources/config/doctrine/Article.orm.xml.skeleton

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping" xsi="http://www.w3.org/2001/XMLSchema-instance" schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+    <!--
+         This file has been generated by the EasyExtends bundle ( https://sonata-project.org/easy-extends )
+
+         References :
+            xsd                  : https://github.com/doctrine/doctrine2/blob/master/doctrine-mapping.xsd
+            xml mapping          : http://www.doctrine-project.org/projects/orm/2.0/docs/reference/xml-mapping/en
+            association mapping  : http://www.doctrine-project.org/projects/orm/2.0/docs/reference/association-mapping/en
+    -->
+    <entity
+        name="Application\Sonata\ArticleBundle\Entity\Article"
+        table="article__article"
+        >
+
+        <id name="id" type="integer" column="id">
+            <generator strategy="AUTO"/>
+        </id>
+
+    </entity>
+</doctrine-mapping>
```

Comment:
Please configure your editor to always add a newline at end of file, it will make the red pictogram disappear.